tests/accounts/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
+from django.urls import path
+
+from maykin_common.accounts.password_reset import PasswordResetView
+
+urlpatterns = [
+    path(
+        "admin/password_reset/",
+        PasswordResetView.as_view(),
+        name="admin_password_reset",
+    ),
+    path("admin/", admin.site.urls),
+]
```

Comment:
I would not define any urls.py here like that, you're messing with the root URL routing of a project and by including the `admin` URLs like that you remove flexibility and create frustration. Don't get in the way of projects.

Instead, document the code snippet for the URLs to include (and update default-project accordingly) so that it can easily be copy-pasted.